tsconfig.base.json

```diff
@@ -13,6 +13,8 @@
     "forceConsistentCasingInFileNames": true,
     "resolveJsonModule": true,
     "allowSyntheticDefaultImports": true,
+    // @todo: Migrate deprecated "importsNotUsedAsValues" to "verbatimModuleSyntax"
```

Comment:
I got this message as well, but it was gone after removing the @types/node dependency